bundles/config/org.eclipse.smarthome.config.xml/config-description-1.0.0.xsd

```diff
@@ -1,57 +1,80 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" 
```

Comment:
In Eclipse, is XML formatter specified somewhere? Currently I only have the java formatter and I can't find anywhere to add the XML...